website/docs/tutorials/guides/dependencies-management.md

```diff
@@ -0,0 +1,91 @@
+:::
+
+## Adding a dependency
+
+To add a dependency, inside project directory, run `protostar install EXTERNAL_DEPENDENCY_REFERENCE`:
+
+```console title="Installing a dependency from link to a repository."
+$ protostar install https://github.com/OpenZeppelin/cairo-contracts
+12:00:00 [INFO] Installing cairo_contracts (https://github.com/OpenZeppelin/cairo-contracts)
+```
+
+
+```console title="'lib' category contains the installed dependency."
+$ tree -L 2
+.
+├── lib
+│   └── cairo_contracts
+├── protostar.toml
+├── src
+│   └── main.cairo
+└── tests
+    └── test_main.cairo
+```
+
+### External dependency reference formats
+
+Protostar supports the following ways of referencing external dependency:
+
+| Format                                | Example                                           |
+| ------------------------------------- | ------------------------------------------------- |
+| `GITHUB_ACCOUNT_NAME/REPO_NAME[@TAG]` | `OpenZeppelin/cairo-contracts@0.1.0`              |
+| `URL_TO_THE_REPOSITORY`               | `https://github.com/OpenZeppelin/cairo-contracts` |
+| `SSH_URI`                             | `git@github.com:OpenZeppelin/cairo-contracts.git` |
+
+### Aliases
+
+Protostar supports installing dependencies under a different name. This allows you to resolve a name conflict, in case of two GitHub users use the same name for their library. In order to install a package under a custom name, run `protostar install EXTERNAL_DEPENDENCY_REFERENCE --name CUSTOM_NAME`. [Updating dependencies](#updating-dependencies) section explains how to refer to installed dependency.
+
+```console title="Installing a dependency under different name."
+$ protostar install https://github.com/OpenZeppelin/cairo-contracts --name open_zeppelin
+10:09:51 [INFO] Installing open_zeppelin (https://github.com/OpenZeppelin/cairo-contracts)
+```
+
+## Installing dependencies after cloning a repository
+
+If you [clone](https://git-scm.com/docs/git-clone) Protostar project using dependencies without `--recurse-submodules` flag, you need to install dependencies using Protostar. Otherwise, your project won't compile and tests will fail. To do so, run `protostar install` in the project directory.
+
+```console title="Protostar will install all submodules from the dependencies directory."
+$ protostar install
+09:37:42 [INFO] Installing cairo_contracts (https://github.com/OpenZeppelin/cairo-contracts)
+```
+
+
+## Updating dependencies
+
+To update:
+
+- a single dependency, run `protostar update LOCAL_DEPENDENCY_REFERENCE`
+- all dependencies, run `protostar update`
+
+`LOCAL_DEPENDENCY_REFERENCE` is a directory name of dependency for example:
+
+```console title="Updating a previously installed dependency."
+$ protostar update cairo_contracts
+10:03:52 [INFO] Package already up to date.
+```
+
+:::note
+You can use [`EXTERNAL_DEPENDENCY_REFERENCE`](#external-dependency-reference-formats) in place of `LOCAL_DEPENDENCY_REFERENCE`.
+:::
```

Comment:
Example for that would be nice, something like:

You can also use EXTERNAL_DEPENDENCY_REFERENCE to update 

```console title="Updating a previously installed dependency."
$ protostar update https://github.com/OpenZeppelin/cairo-contracts
10:03:52 [INFO] Package already up to date.
```